CruCentralCoast/Community/Missions/MissionsTableVC.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  MissionsTableViewController.swift
+//  CruCentralCoast
+//
+//  Created by Landon Gerrits on 6/6/18.
+//  Copyright © 2018 Landon Gerrits. All rights reserved.
+//
+
+import UIKit
+
+class MissionsTableVC: UITableViewController {
```

Comment:
Delete this class since it isn't used anymore. We don't want it to enter the repo